### PR TITLE
OpInfo : embedding_bag

### PR DESCRIPTION
**Pull request by kshitij12345**
Adds OpInfo for `embedding_bag`.

### CODE REVIEW

**Comment by kshitij12345** [comment]
@zou3519 gentle ping :)

**Comment by zou3519** [comment]
Sorry for the delay, will review soon!

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -5921,6 +5921,135 @@ def sample_inputs_dropout(op_info, device, dtype, requires_grad, **kwargs):
...
+                per_sample_weights = make_per_sample_weight(generate_per_sample_weight, idx)
+                yield SampleInput(make_input((M, S)), args=(idx,),
+                                  kwargs={'offsets': offsets, 'mode': mode,
+                                          'per_sample_weights': per_sample_weights})
+
+                idx = make_long_input((S,), low=0, high=M, noncontiguous=True)
+                per_sample_weights = make_per_sample_weight(generate_per_sample_weight, idx)
+                yield SampleInput(make_input((M, S)), args=(idx,),
+                                  kwargs={'offsets': offsets, 'mode': mode,
+                                          'per_sample_weights': per_sample_weights})
```
Comment (zou3519) [inline torch/testing/_internal/common_methods_invocations.py:5958]: Hmm, we should test some offsets with a zero-length bag. E.g. offset = torch.tensor([0, 0, 3])

**Review by zou3519 — APPROVED** [review]
One minor suggestion for a test case, otherwise this LGTM

**Review by zou3519 — APPROVED** [review]

**Comment by kshitij12345** [comment]
@zou3519 Looks like previous failure was due to flakiness.

**Comment by kshitij12345** [comment]
@zou3519 gentle ping :)

**Comment by kshitij12345** [comment]
@zou3519 gentle ping :)

**Pull request closed without merging**